src/rest_api/classes/do_term/widgets/overview.clj

```diff
@@ -0,0 +1,115 @@
+(ns rest-api.classes.do-term.widgets.overview
+  (:require
+   [clojure.string :as str]
+   [rest-api.classes.generic-fields :as generic]
+   [rest-api.formatters.object :as obj :refer [pack-obj]]))
+
+(defn- gene-disease-relevance [gene]
+  (when-let [drhs (:gene/disease-relevance gene)]
+    (for [drh drhs :let [note (:gene.disease-relevance/note drh)
+                         species (:gene.disease-relevance/species drh)]]
+      {:text note
+       :evidence (obj/get-evidence drh)})))
+
+(defn- gene-disease-orthologs [gene]
+  (when-let [ohs (:gene/ortholog gene)]
+    (remove
+      nil?
+      (for [oh ohs :let [sh (:gene.ortholog/species oh)
+
+                         species-id (:species/ncbi-taxonomy sh)]]
+        (if (= species-id 9606)
+          (let [ortholog (:gene.ortholog/gene oh)
+                dhs (:gene/database ortholog)
+                id  (first
+                      (remove
+                        nil?
+                        (for [dh dhs
+                              :let [source-db (:database/id (:gene.database/database dh))
+                                    source-type  (:database-field/id (:gene.database/field dh))]]
+                          (if (and
+                                (= source-db "OMIM")
+                                (= source-type "gene"))
+                            (:gene.database/accession dh)))))]
+            (if (some? id)
```

Comment:
@mgrbyte you are right. I was making a minor point anyway. The truth value of id vs (some? id) is Only different if id is the value `false`. I don't think it's a distinction we care to make.